README.md

```diff
@@ -1,13 +1,39 @@
 # Rails + React + Webpacker + Webpacker-React
 
-
 This is a template Rails 5.1 application, with Webpacker + React.
 
 React + React Router implementation follows Launchpad's established style and organization principles.
 
-If you choose to tackle setup and configuration, feel free to follow our [guide](https://github.com/LaunchPadLab/process).
+Guide is in the works if you wish to tackle setup and configuration.
+<!-- If you choose to tackle setup and configuration, feel free to follow our [guide](https://github.com/LaunchPadLab/process). -->
+
+## Getting Started
+
+  1. `$ git clone git@github.com:LaunchPadLab/react-rails-five-one.git`
+  2. `$ cd react-rails-five-one`
+  3. `$ bundle install && yarn install` 
+
+  This template ships with Todo MVC, which includes a Todos model. 
+  If you do not wish to include this model in your database, delete all migrations related to Todos.
+
+  To start your Rails server, run:
+
+  4. `$ rails s`
+
+  To start your Webpack build, run:
+
+  5. `$ ./bin/webpack-dev-server`
```

Comment:
You should add a `start` script to your `package.json` that runs both of these commands.

Reply:
You can use foreman, that's how it's set up in VS. It's good to have the option to run everything at once, even if you prefer to run them separately in your own dev environment.